Cargo.toml

```diff
@@ -7,13 +7,39 @@ license = "MIT"
 repository = "https://github.com/Valensas/rust-actuator"
 
 [dependencies]
-prometheus = "0.13.3"
+async-trait = "0.1.73"
+rocket = { version = "0.5.0-rc.3", features = ["json"] }
```

Comment:
Neden rc3? 0.5.0 spesifik bir şey mi kullanıyoruz?

Reply:
my turkish is not that good but, if you are asking library version, https://rocket.rs/master as they mentioned here, latest release is this

Reply:
But, it is not released, release candidate.